### PR TITLE
adding netlify logo for open source plan

### PR DESCRIPTION
**Pull request by thegrims**
Having the open-source plan will give the project more build minutes and allow for members other than myself to be added to the site Netlify organization and have control over hosting aspects of the site

https://www.netlify.com/legal/open-source-policy

To qualify for the Open Source plan, a project must adhere to the following criteria:

- Includes a license listed on the Open Source Initiative approved license list or a Creative Commons license that includes “attribution” or places the work in the public domain.
- Features a Code of Conduct at the top level directory of the project repository or prominently in the documentation (with a link in the navigation, footer, or homepage).
- Must feature a link to our service on your main page, or all internal pages. You have two options:
     We have premade badges for your convenience, or
     You may create your own link, which should read “This site is powered by Netlify”, and include a link back to our home page.
- Must not be a commercial project, whether created by a company or an individual. This prohibition includes commercial support and hosting services.

### CODE REVIEW

**Comment by bluSCALE4** [comment]
Would it be better to host the svg's ourselves?

File: src/components/GettingStarted.tsx [hunk 1]
```diff
@@ -122,6 +124,15 @@ export default function GettingStarted(): ReactElement {
         Contributions to the <Link href={urls.repo}>GitHub</Link> repository are
         welcome.
       </p>
+      {prefersDarkMode ? (
+        <a href="https://www.netlify.com">
+          <img src="netlify-dark.svg" alt="Deploys by Netlify" />
+        </a>
+      ) : (
+        <a href="https://www.netlify.com">
```
Comment (zakpatterson) [inline src/components/GettingStarted.tsx:132]: Could be simplified right
```ts
        <a href="https://www.netlify.com">
          <img src={prefersDarkMode ? 'netlify-dark.svg' : 'netlify-light.svg'} alt="Deploys by Netlify" />
        </a>
```

**Comment by zakpatterson** [comment]
Is this the same as getting paid $10 a month or so to promote netlify? I'm confused about why it's different from any other advertising. If someone approached us asking for a link back in exchange for some amount of money we would say no, right? 

I'm not sure I have an opinion yes or no. It's just this one part of the idea doesn't seem right.

**Comment by zakpatterson** [comment]
> If someone approached us asking for a link back in exchange for some amount of money we would say no, right?

I'm admitting this perspective is a bit closed-minded, I think something like [this site](https://gohugo.io/) has a good sponsors footer. If we had a list of people that were willing to sponsor our work, I think it would be fine to recognize them. You outlined the benefits above accurately. I've thought about it again from the other point of view and I think it's fine. This is one service that would be provided for free, and it's good that this company is providing open-source projects free build time, they don't have to do that. 

So sure, I agree this is good.

**Comment by thegrims** [comment]
> > If someone approached us asking for a link back in exchange for some amount of money we would say no, right?
> 
> I'm admitting this perspective is a bit closed-minded, I think something like [this site](https://gohugo.io/) has a good sponsors footer. If we had a list of people that were willing to sponsor our work, I think it would be fine to recognize them. You outlined the benefits above accurately. I've thought about it again from the other point of view and I think it's fine. This is one service that would be provided for free, and it's good that this company is providing open-source projects free build time, they don't have to do that.
> 
> So sure, I agree this is good.

I'm good either way, it would be a nice addition but you're right I don't think we want to be doing a lot of sponsorships. If it would be better I can switch the badge image to a link and or create a footer below the main content to put this in

**Review by zakpatterson — APPROVED** [review]

**Comment by zakpatterson** [comment]
I think this is fine, later we might want to tweak the design a bit to include a footer or some other way to bring more attention to our github/twitter for user engagement too.

**Pull request merged**